src/features/Settings/components/AddressBook.tsx

```diff
@@ -125,6 +125,9 @@ const getSortingFunction = (sortKey: ISortTypes): TSortFunction => {
   }
 };
 
+export const screenIsMobileSized = (): boolean =>
+  window.matchMedia(`(max-width: ${breakpointToNumber(BREAK_POINTS.SCREEN_XS)}px)`).matches;
+
```

Comment:
Same here

Reply:
It has to match the breakpoint for the table. I have simplified.